src/Notification/Notification.module.scss

```diff
@@ -0,0 +1,107 @@
+@import '../mixins.scss';
+@import '../variables.scss';
+
+.wrapper {
+    @include text-normal;
+
+    border-radius: $border-radius;
+    padding: 14px 18px 16px 15px;
+    width: 320px;
+    margin-bottom: 15px;
+}
+
+.default {
+    color: var(--grey-dark-2);
+    background-color: var(--white);
+    fill: var(--black);
+
+    .title {
+        color: var(--black);
+    }
+
+    .closeIcon {
+        fill: var(--grey-dark);
+    }
+}
+
+.header {
+    display: flex;
+    align-items: center;
+}
+
+.closeIcon {
+    cursor: pointer;
+    width: 13px;
+    height: auto;
+}
+
+.titleIcon {
+    margin: 0 15px 0 0;
```

Comment:
`padding: 0 15px` on the `.title` results in the same thing but with one less class